javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,51 @@
+const button = document.querySelector("button");
+const sCounter = document.getElementById("press-s-counter");
+const lCounter = document.getElementById("press-l-counter");
+
+button.addEventListener("click", () => {
+  const input = document.querySelector("input").value;
+  document.addEventListener("keydown", getKey);
+  //take the input as timer
+  setTimeout(() => {
+    if (countS > countL) {
+      alert("player s win");
+      countS = 0;
```

Comment:
you repeat  the following code over and over again . Try to put it in a function like 
`resetGame()` where 
` countS = 0;
   countL = 0;
   sCounter.innerHTML = " ";
   lCounter.innerHTML = " ";
`